src/codeu/chat/server/Model.java

```diff
@@ -134,6 +142,24 @@ public void add(Message message) {
   public StoreAccessor<String, Message> messageByText() {
     return messageByText;
   }
+<<<<<<< HEAD
+
+  public void changeAuthority(ConversationHeader conversation, Uuid targetUser, byte authorityByte)
+  {
+    LinkedHashMap<Uuid,Byte> userAuthority;
+    if(authority.get(conversation.id) == null)
+    {
+       authority.put(conversation.id, new LinkedHashMap<>());
```

Comment:
Clever. But use `conversation` not `conversation.id`, really, it'll help in the long run!